lib/Models/CatalogItem.js

```diff
@@ -167,8 +167,18 @@ var CatalogItem = function(terria) {
      */
     this.maximumShownFeatureInfos = terria.configParameters.defaultMaximumShownFeatureInfos;
 
+    /**
+     * Options for formatting current time and timeline tic labels. See tests for examples.
```

Comment:
These comments get turned into documentation for the data source editor, which is used by non-developers.  So referring them to the tests doesn't really work, you need to enumerate the options here.  Sorry.